src/Razor/Microsoft.AspNetCore.Razor.Language/src/RazorLanguageVersion.cs

```diff
@@ -41,6 +43,11 @@ public static bool TryParse(string languageVersion, out RazorLanguageVersion ver
                 version = Experimental;
                 return true;
             }
+            else if (languageVersion == "5.0")
```

Comment:
You need to add this equivalent to the tooling repo too: https://github.com/dotnet/aspnetcore-tooling/blob/671d0792fb3fb587766a5800fafb0eee2d8f43cd/src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackRazorConfiguration.cs#L37-L39